### PR TITLE
Add intensity slider in heatmap

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Add intensity slider in heatmap

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/components/input/SliderInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+function Track<T extends number | number[]>(
+    props: HTMLPropsWithRefCallback<HTMLSpanElement>,
+    state: { index: number; value: T },
+) {
+    const { index } = state;
+    return (
+        <div
+            {...props}
+            // index={index}
```
Comment (samshara) [inline app/components/input/SliderInput/index.tsx]: let's remove this

File: app/components/input/SliderInput/index.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        label,
+        minDistance,
+        hideValues,
+        hideMarks,
+    } = props;
+
+    const marks = Array.from(
+        { length: max - min + 1 },
+        (_, i) => i + min,
+    );
```
Comment (samshara) [inline app/components/input/SliderInput/index.tsx]: we could memoise this

File: app/views/NewExploreDeep/EntriesContent/EntriesHeatMap/styles.css [hunk 3]
```diff
@@ -2,3 +2,16 @@
     width: auto;
     height: 50em;
 }
+
+.controls {
+    position: absolute;
+    right: 0;
+    bottom: var(--dui-spacing-large);
+    background-color: rgba(255, 255, 255, 0.6);
```
Comment (tnagorra) [inline app/views/NewExploreDeep/EntriesContent/EntriesHeatMap/styles.css]: Let's use css variables

File: app/views/NewExploreDeep/index.tsx [hunk 4]
```diff
@@ -348,9 +451,53 @@ function NewExploreDeep(props: Props) {
...
+        excludeEntryLessThan: filters?.excludeProjectsLessThan,
+    }), [filters]);
+
+    const {
+        previousData: previousEntriesTimeseriesData,
+        data: entriesTimeseriesData = previousEntriesTimeseriesData,
+    } = useQuery<EntriesCountTimeseriesQuery, EntriesCountTimeseriesQueryVariables>(
+        ENTRIES_COUNT_TIMESERIES,
+        {
+            skip: printPreviewMode ? false : (activeView !== 'entries'),
```
Comment (tnagorra) [inline app/views/NewExploreDeep/index.tsx]:
```suggestion
            skip: !printPreviewMode && activeView !== 'entries',
```

File: app/views/NewExploreDeep/index.tsx [hunk 5]
```diff
@@ -348,9 +451,53 @@ function NewExploreDeep(props: Props) {
...
+        },
+    );
+
+    const {
+        previousData: previousProjectsTimeseriesData,
+        data: projectsTimeseriesData = previousProjectsTimeseriesData,
+    } = useQuery<ProjectCountTimeseriesQuery, ProjectCountTimeseriesQueryVariables>(
+        PROJECT_COUNT_TIMESERIES,
+        {
+            skip: printPreviewMode ? false : (activeView !== 'projects'),
```
Comment (tnagorra) [inline app/views/NewExploreDeep/index.tsx]:
```suggestion
            skip: !printPreviewMode && activeView !== 'projects',
```

File: app/views/NewExploreDeep/index.tsx [hunk 6]
```diff
@@ -348,9 +451,53 @@ function NewExploreDeep(props: Props) {
...
+    );
+
+    const {
+        previousData: previousEntriesMapData,
+        data: entriesMapData = previousEntriesMapData,
+        loading: loadingMapAndLeadData,
+    } = useQuery<EntriesMapDataQuery, EntriesMapDataQueryVariables>(
+        ENTRIES_MAP_DATA,
+        {
+            skip: printPreviewMode ? false : (activeView !== 'entries'),
```
Comment (tnagorra) [inline app/views/NewExploreDeep/index.tsx]:
```suggestion
            skip: !printPreviewMode && activeView !== 'entries',
```

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by tnagorra — APPROVED** [review]

**Pull request merged**